time stamp
uint32 robot_id
string add_info
float32 cost
float32[] params
time start_t
float32[] start_q
time end_t
float32[] end_q